packages/@okta/vuepress-site/docs/reference/api/trusted-origins/index.md

```diff
@@ -121,7 +124,207 @@ curl -X POST \
 }
 ```
 
+#### Valid request example with iFrame embedding
+
+<ApiLifecycle access="ea" />
+
+Creates a new Trusted Origin for iFrame embedding of an Okta resource within that origin. In this example, the type of Okta resource is both the Okta End-User Dashboard and the Okta sign-in page.
+
+```bash
+curl -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "name": "New Trusted Origin",
+  "origin": "http://example.com",
+  "scopes": [
+    {
+      "type": "CORS"
+    },
+    {
+      "type": "REDIRECT"
+    },
+    {
+      "type": "IFRAME_EMBED",
+      "allowedOktaApps": [“OKTA_ENDUSER”]
+    }
+  ]
+}' "https://${yourOktaDomain}/api/v1/trustedOrigins"
+```
+
+Creates a new Trusted Origin for iFrame embedding of an Okta resource within that origin. In this example, the Okta resource is the Okta sign-in page.
```

Comment:
Are the CORS and REDIRECT scopes required for this example?  If not, since the example pivots around iFrame embedding, maybe we could omit them.  Applies to examples below.

Reply:
i have updated the example